frontend/src/container/FeedItemWithPreview.jsx

```diff
@@ -153,14 +172,17 @@ export class FeedItemWithPreview extends React.Component {
             id={props.content.id}
             invalidMentionList={state.invalidMentionList}
             loggedUser={props.user}
+            memberList={props.memberList}
             newComment={state.newComment}
             newCommentAsFileList={state.newCommentAsFileList}
             onChangeNewComment={this.handleChangeNewComment}
             onRemoveCommentAsFile={this.handleRemoveCommentAsFile}
-            onValidateCommentFileToUpload={this.handleAddCommentAsFile}
+            onClickDeleteComment={comment => this.handleClickDeleteComment(comment)}
+            onClickEditComment={comment => this.handleClickEditComment(comment)}
```

Comment:
you can pass the function as reference here
```suggestion
            onClickDeleteComment={this.handleClickDeleteComment}
            onClickEditComment={this.handleClickEditComment}
```
You only need to redefine parameters when you want to change them

Reply:
will fix